### PR TITLE
Modularize import statement

### PR DESCRIPTION
**Pull request by erickcelio**
Just made modularization on the backend.

## Description

 I use the example in #87, so I created a root import @server appoint to server folder, now you can use like @server/core, @server/socket.

## Related Issue

#86 

## Motivation and Context

It made the imports more readable.

## How Has This Been Tested?

Q&A (manual testing)

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by naknode** [comment]
Thanks @erickcelio, I'll take a closer look over soon.

File: server/Delaford.js [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-
-// Default entities
-const NPC = require('./core/npc');
-const Map = require('./core/map');
-const Item = require('./core/item');
-
-// Tools and utility
-const Authentication = require('./player/authentication');
+import uuid from 'uuid/v4';
+import world from '@server/core/world';
```
Comment (naknode) [inline server/Delaford.js]: If you could keep the comments and have the same structure as before (`// Default entities, // Tools etc`) that would be great.

File: server/core/data/query.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import { wearableItems, general } from 'root/core/data/items';
-import { foregroundObjects } from 'root/core/data/foreground';
+import { general, wearableItems } from '@server/core/data/items';
+
```
Comment (naknode) [inline server/core/data/query.js]: Remove blank line, please.

File: server/player/action.js [hunk 3]
```diff
@@ -1,23 +1,22 @@
 import UI from 'shared/ui';
+import World from '@server/core/world';
```
Comment (naknode) [inline server/player/action.js]: Any reason why you capitalized the `w` here?

**Comment by naknode** [comment]
I can fix the smaller issues later. Thank you!

**Comment by erickcelio** [comment]
> I can fix the smaller issues later. Thank you!

I will make a PR for these things sorry about that :smile:

**Pull request merged**